### PR TITLE
Alex/36 connect featured charity api

### PR DESCRIPTION
**Pull request by alexandrashaw10**

### CODE REVIEW

File: app/src/main/java/com/example/una/adapters/CategoryAdapter.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package com.example.una;
+package com.example.una.adapters;
```
Comment (chiped) [inline app/src/main/java/com/example/una/adapters/CategoryAdapter.java]: 👍

**Comment by alexandrashaw10** [comment]
image

File: app/src/main/java/com/example/una/fragments/HomePageFragment.java [hunk 2]
```diff
@@ -139,6 +163,14 @@ public void onFailure(int statusCode, Header[] headers, String responseString, T
         });
     }
 
+    private RequestParams buildParams() {
```
Comment (chiped) [inline app/src/main/java/com/example/una/fragments/HomePageFragment.java]: 👍

File: app/build.gradle [hunk 3]
```diff
@@ -68,6 +68,9 @@ dependencies {
     // Firebase dependencies
     implementation 'com.google.firebase:firebase-core:17.0.0'
     implementation 'com.firebaseui:firebase-ui-auth:4.3.1'
+
+    // Gson Dependencies
+    implementation 'com.google.code.gson:gson:2.8.5'
```
Comment (chiped) [inline app/build.gradle]: do you use gson?

**Comment by alexandrashaw10** [comment]
Issue #36

**Pull request merged**